website/src/components/Changelog/GUI.tsx

```diff
@@ -13,12 +13,17 @@ export default function GUI({ title }: { title: string }) {
   return (
     <Entries href={href} arches={arches} title={title}>
       {/*
-      <Entry version="1.2.0" date={new Date("Invalid date")}>
+      <Entry version="1.1.12" date={new Date("Invalid date")}>
         <ul className="list-disc space-y-2 pl-4 mb-4">
           <ChangeItem pull="6226">
             Fixes a bug where clearing the log files would delete the current files, preventing logs from being written.
           </ChangeItem>
         </ul>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+          <ChangeItem pull="6276">
+            Fixes a bug where relayed connections failed to establish after an idle period.
+          </ChangeItem>
+        </ul>
```

Comment:
```suggestion
          <ChangeItem pull="6276">
            Fixes a bug where relayed connections failed to establish after an idle period.
          </ChangeItem>
```

Reply:
repeated the `<ul>` here I think by accident

Reply:
Ah yes, my HTML-fu is a bit dated.